socket/net.go

```diff
@@ -47,6 +47,12 @@ func ParseAddress(input string) (network, address, host string, err error) {
 		return
 	}
 
+	if strings.HasPrefix(input, "http:") {
+		network = "unix"
```

Comment:
Not sure why this block is needed?

Reply:
Sorry for the typo, accidentally click ctrl +v and miss when committing.

amended commit.